src/pkg/ThirdPartyNotices.txt

```diff
@@ -0,0 +1,31 @@
+This Microsoft .NET Library may incorporate components from the projects listed
```

Comment:
corefx has the pkg directory directory under root. Should we follow the same directory structure?